main/acle.md

```diff
@@ -10132,6 +10184,9 @@ Multi-vector convert to/from floating-point.
 
   // Variants are also available for _f32[_u32_x4], _s32[_f32_x4] and _u32[_f32_x4]
   svfloat32x4_t svcvt_f32[_s32_x4](svint32x4_t zn) __arm_streaming;
+
+  // Only if __ARM_FEATURE_SME_F16F16 != 0
+  svfloat32x2_t svcvt_f32[_f16_x2](svfloat16_t zn) __arm_streaming;
```

Comment:
Wouldn't naming this `svcvt_f32[_x2_f16]` be more consistent as it is output that gets duplicated ?

Reply:
The convention so far has been that the positions of the suffixes are fixed relative to each other.  So type suffixes always come before predication, and always come before x2/x4/etc.

The suffixes are documented as follows:

*   Intrinsic functions have a `_x2` or `_x4` suffix if the
    function\'s widest type is a vector tuple of 2 or 4 data vectors
    and the function operates purely on vectors, not on the matrix array or
    tile slices. The suffix is only present on overloaded names if it cannot
    be inferred from arguments.